activesupport/lib/active_support/multibyte/chars.rb

```diff
@@ -5,6 +5,11 @@
 require "active_support/core_ext/string/behavior"
 require "active_support/core_ext/module/delegation"
 
+ActiveSupport.deprecator.warn(
```

Comment:
I was just running against head, and most applications will currently hit this deprecation because it is required by rails here:
https://github.com/rails/rails/blob/aaa111f4c39b441b45d8b1b1408ee0052ed27d9e/activerecord/lib/active_record/connection_adapters/abstract/quoting.rb#L4-L5

Reply:
Ah damn, thank you for letting me know.